testsuite/tests/ui/api_docs/test_autocomplete.py

```diff
@@ -18,14 +17,14 @@ def test_autocomplete(navigator, service, testconfig):
     token = testconfig["threescale"]["admin"]["token"]
 
     api_docs_page = navigator.navigate(APIDocsView)
-    endpoint = api_docs_page.endpoint("Service Read")
-    endpoint.fill(rawobj.ApiDocParams(token))
-    endpoint.get_param("service_ids").click()
+    endpoint = api_docs_page.endpoint("GET", "/admin/api/services/{id}.xml")
+    endpoint.set_param("access_token", token)
+    endpoint.set_param("id", service["name"])
 
-    product_id_element = api_docs_page.get_id_input_by_name(service["name"])
-    auto_comp_prod_id = product_id_element.text
-    product_id_element.click()
-    assert auto_comp_prod_id == str(service["id"])
+    # endpoint.get_param("id").click()
+    # product_id_element = api_docs_page.get_id_input_by_name(service["name"])
```

Comment:
I think that these shouldn't be here.

Reply:
You are right. We need to discuss if this is the correct approach how to test autocomplete